app/views/pkg/versions/version_row.mustache

```diff
@@ -6,13 +6,17 @@
   <td><strong><a href="/packages/{{package}}/versions/{{version}}">{{version}}</a></strong></td>
   <td>{{short_created}}</td>
   <td class="documentation">
-    <a href="{{& documentation_url}}"
+    <a class="version-table-icon" href="{{& documentation_url}}"
```

Comment:
Maybe only show the documentation link if `dartdoc_ok`?

Reply:
SGTM, will update.